src/cosmosdb-preview/azext_cosmosdb_preview/commands.py

```diff
@@ -161,3 +162,16 @@ def load_command_table(self, _):
 
     with self.command_group('cosmosdb table', cosmosdb_table_sdk, client_factory=cf_table_resources) as g:
         g.custom_command('retrieve-latest-backup-time', 'cli_table_retrieve_latest_backup_time', is_preview=True)
+
+    # Data Transfer Service
+    cosmosdb_data_transfer_job = CliCommandType(
+        operations_tmpl='azext_cosmosdb_preview.vendored_sdks.cosmodb.operations._data_transfer_jobs_operations#DataTransferJobsOperations.{}',
+        client_factory=cf_data_transfer_job
+    )
+
+    with self.command_group('cosmosdb dts', cosmosdb_data_transfer_job, client_factory=cf_data_transfer_job) as g:
+        g.custom_command('export', 'cosmosdb_data_transfer_export_job')
+        g.custom_command('import', 'cosmosdb_data_transfer_import_job')
+        g.custom_command('copy', 'cosmosdb_data_transfer_copy_job')
+        g.custom_command('list', 'cosmosdb_dts_list')
+        g.custom_show_command('show', 'cosmosdb_dts_show')
```

Comment:
```suggestion
        g.command('list', 'list_by_database_account')
        g.show_command('show', 'get')
```
No need to use custom command, the function from SDK can be called directly